.maintain/chaostest/src/hypervisor/utils/json-rpc.js

```diff
@@ -0,0 +1,24 @@
+const request = require('request')
+
+const getChainBlockHeight = (url, port) => new Promise((resolve, reject) => {
+  let options = {
+    url: url + ':' + port,
+    method: 'post',
+    headers:
+	{'content-type': 'application/json'},
+    body: JSON.stringify({id: 1, jsonrpc: '2.0', method: 'chain_getBlock'}),
```

Comment:
I think it might be a good idea to introduce `@polkadot/api` already, to see how it fits in with other global objects such as `CONFIG` and `k8s`.